### PR TITLE
Add known issues doc

### PR DESCRIPTION
**Pull request by xing-yang**
Add a doc about known issues.

### CODE REVIEW

File: docs/known_issues.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Velero Plugin for vSphere Known Issues
+
+This section lists the major known issues with Velero Plugin for vSphere. For a complete list of issues, check the [Github issues](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/issues) page. If you notice an issue not listed in Github issues page, please file an issue on the Github repository.
+
+## Backup and restore in maintenance mode
+
+1. Backup or Restore operation while the host (which has no PVC/PV/Pods associated with the workload) in maintenance mode - the backup/restore will work fine.
```
Comment (lintongj) [inline docs/known_issues.md:7]: host --> ESXi host

File: docs/known_issues.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+2. Backup or Restore operation while the host (which has PVC/PV/Pods associated with the workload) in maintenance mode - the backup/restore is expected to fail as the volumes will be inaccessible during this period.
```
Comment (lintongj) [inline docs/known_issues.md:9]: host --> ESXi host

Comment (xing-yang) [reply docs/known_issues.md:9]: Done

File: docs/known_issues.md (hunk 1 above)
Comment (xing-yang) [reply docs/known_issues.md:7]: I'll remove point 1.

File: docs/known_issues.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Velero Plugin for vSphere Known Issues
+
+This section lists the major known issues with Velero Plugin for vSphere. For a complete list of issues, check the [Github issues](https://github.com/vmware-tanzu/velero-plugin-for-vsphere/issues) page. If you notice an issue not listed in Github issues page, please file an issue on the Github repository.
+
+## vSphere CSI driver config file
+
+On a Vanilla cluster, Velero Plugin for vSphere expects the key `csi-vsphere.conf` being presented in the `data` field of the secret `vsphere-config-secret`. If it is not present, backup will fail.
```
Comment (swgupta-vmw) [inline docs/known_issues.md:7]: Can we mention that this key and secret names are the default when vsphere csi is installed.

**Review by swgupta-vmw — APPROVED** [review]
Change look good. Added 1 comment.

File: docs/known_issues.md (hunk 3 above)
Comment (xing-yang) [reply docs/known_issues.md]: Done

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/435/

**Pull request merged**